### DISCUSSION TITLE
windows server 2008 r2

### DISCUSSION THREAD

**Discussion by yf2468** [AlistGo/alist#5898] [General]
My server is Windows server 2008 r2.
I run "alist server" at command line,  but nothing happened. No error message.

**Comment by yf2468** [comment]
windows2008能否安装，有人可以解答吗？

**Reply by allanghub** [reply]
我安装的是3.25.1版本的，再往后就出错退出了

**Comment by anwen-anyi** [comment]
AList v3.30.0版本开始将不再支持Win7(server2008)，因为Go不支持了,
- 点击查看详细说明:https://github.com/golang/go/issues/64622
